msal/tests/Test.MSAL.NET.Unit.net45/CryptographyTests.cs

```diff
@@ -36,6 +36,7 @@
 
 namespace Test.MSAL.NET.Unit
 {
+#if !WINDOWS_APP // not available on UWP
```

Comment:
suggestion:  can we setup additional defines somewhere globally (like in the msal csproj?) where we define a feature #define (e.g. MSAL_IS_USERNAME_PASSWORD_AVAILABLE) and we can use these here?  We already decide the feature directories to include within the csproj and it would be great if we could centrally manage those with a define and drive the unit tests against them.  I just see it being hard to get all of the unit tests' #defines maintained properly as I see the #if !ios && !android && !windows_app in this PR.

Reply:
Yes, very good idea. In fact this is how a lot of other multi platform libraries are doing it - they do conditional compilation based on feature flags not on platform flags. I'll raise an issue.